DigitalLearningSolutions.Data.Tests/Helpers/CourseContentTestHelper.cs

```diff
@@ -72,11 +72,13 @@ public void UpdateSystemRefreshed(int candidateId, int customisationId, bool sys
                 new { candidateId, customisationId, systemRefreshed }
             );
         }
-
-        public bool IsApproximatelyNow(DateTime timeToCheck)
+        
+        public bool IsApproximatelyNow(DateTime? timeToCheck)
```

Comment:
@benjimarshall used `Should().BeCloseTo` in one of his recent PRs: https://github.com/TechnologyEnhancedLearning/DLSV2/pull/52/files. That's a bit neater than using our own helper method so probably worth using in your tests as well? 🙂

Reply:
The helper method currently uses a 2 minute window but should I change to 10 seconds for consistency?

Reply:
It depends on the test. If we're checking that something is close to now then I think using two minutes is a good idea because we don't want the test to break if it for some reason takes a bit longer to run the test.